src/playbook.yaml

```diff
@@ -14,7 +14,7 @@
         executable: pip3
 
     - name: Install dependencies
-      shell: "yes | aptdcon --hide-terminal --install gdebi-core zlib1g-dev libclang-dev libssl-dev libffi-dev libcurl4-openssl-dev libapparmor1 libssl1.0.0 libxml2-dev libjpeg-dev libpng-dev libcairo2-dev libxt-dev bzip2 unixodbc unixodbc-devel"
+      shell: "yes | aptdcon --hide-terminal --install \"gdebi-core zlib1g-dev libclang-dev libssl-dev libffi-dev libcurl4-openssl-dev libapparmor1 libssl1.0.0 libxml2-dev libjpeg-dev libpng-dev libfreetype6-dev libtiff5-dev libcairo2-dev libxt-dev bzip2 unixodbc unixodbc-devel\""
```

Comment:
might be cleaner using single quote instead, like this https://sagebionetworks.slack.com/archives/CEFQD0KU1/p1592872598448200

Reply:
Good point.